### DISCUSSION TITLE
Setting the textCapitalization and keyboardType properties

### DISCUSSION THREAD

**Discussion by SarvagyaVaish** [ueman/feedback#139] [Q&A] [answered]
Is it possible to set the following properties to customize the UX of the input field?
- [textCapitalization](https://api.flutter.dev/flutter/material/TextField/textCapitalization.html)
- [keyboardType](https://api.flutter.dev/flutter/material/TextField/keyboardType.html)

**Comment by ueman** [comment, marked as answer]
You can provide  a custom [feedbackBuilder](https://pub.dev/documentation/feedback/latest/feedback/BetterFeedback/feedbackBuilder.html) which you can customize to your liking. The default can currently not be customized. Does this help?
